schemas/com.snowplowanalytics.travel/destination_filter/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.travel",
+    "name": "destination_filter",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Event for filtering destination specific information",
+  "properties": {
+    "page_type": {
+      "type": "string",
+      "description": "Page type"
+    },
+    "action_type": {
```

Comment:
What is a filter action and why do only destination_filters have them and not experience_filters?
Could they be the same schema with a field to discriminate them?